docs/conf.py

```diff
@@ -42,7 +42,7 @@ def get_version(*file_paths):
 VERSION = get_version('../edx_toggles', '__init__.py')
 
 # Configure Django for autodoc usage
-settings.configure()
+os.environ['DJANGO_SETTINGS_MODULE'] = 'test_settings'
```

Comment:
**Reviewer Note:** Should we bring this into cookiecutter? See PR description for details on what this fixed.

Reply:
Yes please

Reply:
Ok. I'll do this after I squash/merge the fix for this.

Reply:
Is this change only relavant to IDAs or also to django apps?

Reply:
Are you asking because of the potential cookiecutter PR? If so, let's have the conversation there: https://github.com/edx/edx-cookiecutters/pull/109  Although, I will note, that I just found the `settings.configure()` line in one place, so that is all I updated.